### ISSUE TITLE
add test mode for multiple paths

### ISSUE THREAD

**Issue by KonradHoeffner** [hitontology/gui#42]
Using a get parameter display a reasonable estimation of the maximally used paths for each connected node pair to check if the new multi path display always creates a resonable edge clone.

**Comment by KonradHoeffner** [comment]
Seven concurrent paths between two given nodes are the maximum that I observed in a quick test so thats how many are shown in the test page now.

![Screenshot from 2024-04-05 14-37-47](https://github.com/hitontology/gui/assets/839577/34cdab82-c293-454a-bda7-2e2e44f6d3ef)

There are a few problems visible here:

* Vertical separation is too large between different classifieds next to each other, This could be fixed with reducing separation in the y axis. Alternatively we could also (selectively) apply the complex path treatment to simple paths or only to those above a certain separation.  
* SoftwareProduct attribute edges overlap each other heavily.

However in practice those problems haven't actually occurred. For example there is only one path each from one given Software Product attribute to another, which is the only way to get both in one path. Similarily, different classifieds don't seem to be connected that often in parallel paths.

So I will mark this issue as completed but if problems show up in the future regarding overlap / paths not being connected we can create a new issue in the future and maybe link back to this.
